### PR TITLE
[usecase-linux][xwalk-3714] Add usecase for window size

### PR DESCRIPTION
**Pull request by kangxu**
Impacted tests(approved): new 1, update 0, delete 0
Unit test platform: [Linux]
Unit test result summary: pass 1, fail 0, block 0

### CODE REVIEW

File: usecase/usecase-wrt-linux-tests/res/testapp/org.windowsize.allmembersspecified/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+-->
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Window Size Test: normal usage with all members specified</title>
+  </head>
+  <body>
+    <p>Test app should take up part of the available display area, but not provide any navigation buttons like "back". The height of test app can be shrinked, but can't be expanded.  Instead the width of test app can't be shrinked, but can be expanded.</p>
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/res/testapp/org.windowsize.allmembersspecified/index.html]: I don't understand what do you mean here.

File: usecase/usecase-wrt-linux-tests/res/testapp/org.windowsize.xwalkboundsfullscreen/index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+-->
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Window Size Test: fullscreen display mode override xwalk-bounds</title>
+  </head>
+  <body>
+    <p>Test app should take up entirety of the available display area.</p>
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/res/testapp/org.windowsize.xwalkboundsfullscreen/index.html]: 'entirety of the available display area' means 'full screen'

File: usecase/usecase-wrt-linux-tests/samples/WindowSize/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## Usecase Design
+
+This sample demonstrates window size support in manifest.json.
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/WindowSize/README.md]: Please add the feature link

File: usecase/usecase-wrt-linux-tests/samples/WindowSize/index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    <div class="content">
+      <h2>Test Steps:</h2>
+      <ol>
+        <li>Install the webapp 'allmembersspecified.deb' and launch it</li>
+        <li>Install the webapp 'widthheight.deb' and launch it</li>
+        <li>Install the webapp 'xwalkboundsfullscreen.deb' and launch it</li>
+      </ol>
+      <h2>Expected Result:</h2>
+      <ol>
+        <li>Test app should take up part of the available display area, but not provide any navigation buttons like "back". The height of test app can be shrinked, but can't be expanded. Instead the width of test app can't be shrinked, but can be expanded.</li>
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/WindowSize/index.html]: Please don't mix up the user interactions and expected results.

File: usecase/usecase-wrt-linux-tests/res/testapp/org.windowsize.allmembersspecified/index.html (hunk 1 above)
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/res/testapp/org.windowsize.allmembersspecified/index.html]: Please don't mix user interaction (steps) and expected results.

**Comment by kangxu** [comment]
@zqzhang 
I have updated it following your comments

**Pull request merged**